LSP.sublime-settings

```diff
@@ -49,8 +49,7 @@
   // Show errors and warnings count in the status bar
   "show_diagnostics_count_in_view_status": false,
 
-  "show_ui_blocking_message": true, // Show UI blocking modals or the full console message and short status notification
-  "show_ui_blocking_err_message": true, // Show UI blocking error modals or the full console message and short status notification
+  "suppress_error_dialogs": false, // Replace UI blocking modals with the full console message and a short status notification
```

Comment:
- Follow conventions in the file (the way the comment is placed)
- Find a better place for the option - right now it's inserted in the middle of diagnostics-related options.

Reply:
moved to others